.github/gke_versions.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
in the CNP repo I see 1.27 at the top of the gke_versions file.
Noting to double-check....